### PR TITLE
Test for report_changes suite

### PR DESCRIPTION
**Pull request by Nicogp**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/22760|

## Description

Added report_changes suite to FIM tests.

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux
  - [x] MacOS
  - [x] Windows

### CODE REVIEW

File: tests/integration/test_fim/test_files/test_report_changes/test_diff_size_limit.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+from wazuh_testing.modules.fim.patterns import DIFF_MAXIMUM_FILE_SIZE, ERROR_MSG_MAXIMUM_FILE_SIZE_EVENT, ERROR_MSG_WRONG_VALUE_MAXIMUM_FILE_SIZE
+from wazuh_testing.tools.monitors.file_monitor import FileMonitor
+from wazuh_testing.utils.callbacks import generate_callback
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+
+from . import TEST_CASES_PATH, CONFIGS_PATH
+
+
+# Marks
+pytestmark = [pytest.mark.tier(level=1)]
```
Comment (TomasTurina) [inline tests/integration/test_fim/test_files/test_report_changes/test_diff_size_limit.py:82]: Add agent, linux, win32 and darwin marks. Do this in all tests where missing.

Comment (Nicogp) [reply tests/integration/test_fim/test_files/test_report_changes/test_diff_size_limit.py]: Done!

File: tests/integration/test_fim/test_files/test_report_changes/test_disk_quota_default.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+suite: files_report_changes
+
+targets:
+    - agent
+
+daemons:
+    - wazuh-syscheckd
+
+os_platform:
```
Comment (TomasTurina) [inline tests/integration/test_fim/test_files/test_report_changes/test_disk_quota_default.py:28]: Validate that the `os_platform` and `os_version` lists are correct in all tests. For example, here is missing `darwin`.

Comment (Nicogp) [reply tests/integration/test_fim/test_files/test_report_changes/test_disk_quota_default.py:28]: Done!

**Review by TomasTurina — APPROVED** [review]

**Pull request merged**